integration/src/integration/java/pl/allegro/tech/hermes/integration/env/HermesIntegrationEnvironment.java

```diff
@@ -51,6 +51,8 @@
 
     @BeforeSuite
     public void prepareEnvironment(ITestContext context) throws Exception {
+        System.setProperty("java.security.auth.login.config", "./kafka_server_jaas.conf");
```

Comment:
This property is related with kafka, therefore I think it will be better to move it into KafkaStarter.